cypress/e2e/signIn.cy.js

```diff
@@ -10,7 +10,7 @@ const homePage = new HomePageObject();
 describe('Sign In page', () => {
   let user;
 
-  before(() => {
+  beforeEach(() => {
     cy.task('db:clear');
```

Comment:
It's not recommended to clear the database before each test. It might cause performance issues. Consider resetting the state in the database only when necessary.

Reply:
done